src/qt/overviewpage.cpp

```diff
@@ -283,7 +283,7 @@ void OverviewPage::on_btnWebsite_pressed()
 
 void OverviewPage::on_btnBX_pressed()
 {
-    QDesktopServices::openUrl(QUrl("https://www.gridcoinstats.eu/block/"));
+    QDesktopServices::openUrl(QUrl("https://www.gridcoinstats.eu/block/#pk_campaign=GridcoinWallet&pk_kwd=" + QString::fromStdString(FormatFullVersion())));
```

Comment:
@tomasbrod I wasn't the one doing the change in the source so I don't think it's my place to do a PR, but I hear you 😁